hscontrol/templates/apple.go

```diff
@@ -0,0 +1,240 @@
+package templates
+
+import (
+	"fmt"
+
+	"github.com/chasefleming/elem-go"
+	"github.com/chasefleming/elem-go/attrs"
+)
+
+func HeadscaleApple(url string) *elem.Element {
+	return elem.Html(nil,
+		elem.Head(attrs.Props{
+			attrs.Lang: "en",
+		},
+			elem.Meta(attrs.Props{
+				attrs.Charset: "UTF-8",
+			}),
+			elem.Meta(attrs.Props{
+				attrs.HTTPequiv: "X-UA-Compatible",
+				attrs.Content:   "IE=edge",
+			}),
+			elem.Meta(attrs.Props{
+				attrs.Name:    "viewport",
+				attrs.Content: "width=device-width, initial-scale=1.0",
+			}),
+			elem.Title(nil,
+				elem.Text("headscale - Apple")),
+			elem.Style(nil,
+				elem.CSS(`
+      body {
```

Comment:
Lets make this use the elem-go style syntax and reuse it between `Windows` and `Apple`.

Reply:
I thought if I do that, It will be redundant for some elements, like the style props should be passed to each header element. Let me do it the way you suggested and I'll let you know